extensions/tailscale/src/toggle.tsx

```diff
@@ -0,0 +1,42 @@
+import { showFailureToast } from "@raycast/utils";
+import { getErrorDetails, getStatus } from "./shared";
+import Disconnect from "./disconnect";
+import Connect from "./connect";
+import { showToast, Toast } from "@raycast/api";
+
+export default async function Toggle() {
+  let subtitle = "Tailscale";
+  try {
+    const isConnected = await isTailscaleConnected();
+
+    if (isConnected) {
+      // Disconnect
+      await showToast({
+        style: Toast.Style.Animated,
+        title: "Disconnecting",
+      });
+
+      await Disconnect();
+    } else {
+      // Connect
+      await showToast({
+        style: Toast.Style.Animated,
+        title: "Connecting",
+      });
+
+      await Connect();
+    }
+  } catch (err) {
+    await showFailureToast(err, { title: "Failed to toggle connection" });
+    subtitle = getErrorDetails(err, "").title;
```

Comment:
This doesn't show the actual HUD, right?
And probably the subtitle above should just be a variable in the catch part, it's not used anywhere else as that's handled by each function already.

I think you are missing a line:
```js
showHUD(`Unable to connect: ${subtitle}`);
```